WebRoot/messages/ZmMsg_ja.properties

```
@@ -3418,6 +3418,8 @@ theme-venice = \u30d9\u30cb\u30b9
 theme-waves = \u30a6\u30a7\u30fc\u30d6
 tiffImage = TIFF\u30a4\u30e1\u30fc\u30b8
 time = \u6642\u9593
+timeInMailListChangeRestartHide = \u4eca\u3059\u3050\u30a2\u30d7\u30ea\u30b1\u30fc\u30b7\u30e7\u30f3\u3092\u30ea\u30ed\u30fc\u30c9\u3057\u3066\u3001\u30e1\u30fc\u30ea\u30f3\u30b0 \u30ea\u30b9\u30c8\u306e\u6642\u9593\u3092\u975e\u8868\u793a\u306b\u3057\u307e\u3059\u304b?
+timeInMailListChangeRestartShow = \u4eca\u3059\u3050\u30a2\u30d7\u30ea\u30b1\u30fc\u30b7\u30e7\u30f3\u3092\u30ea\u30ed\u30fc\u30c9\u3057\u3066\u3001\u30e1\u30fc\u30ea\u30f3\u30b0 \u30ea\u30b9\u30c8\u306b\u6642\u9593\u3092\u8868\u793a\u3057\u307e\u3059\u304b?
```

Please update translations for Japanese language.
```suggestion
timeInMailListChangeRestartHide = \u30e1\u30fc\u30eb\u4e00\u89a7\u306b\u65e5\u6642\u3092\u8868\u793a\u3057\u306a\u3044\u305f\u3081\u306b\u4eca\u3059\u3050\u30d6\u30e9\u30a6\u30b6\u3092\u30ea\u30ed\u30fc\u30c9\u3057\u307e\u3059\u304b\uff1f
timeInMailListChangeRestartShow = \u30e1\u30fc\u30eb\u4e00\u89a7\u306b\u65e5\u6642\u3092\u8868\u793a\u3059\u308b\u305f\u3081\u306b\u4eca\u3059\u3050\u30d6\u30e9\u30a6\u30b6\u3092\u30ea\u30ed\u30fc\u30c9\u3057\u307e\u3059\u304b\uff1f
```

(memo)
```
timeInMailListChangeRestartHide = メール一覧に日時を表示しないために今すぐブラウザをリロード
しますか？
timeInMailListChangeRestartShow = メール一覧に日時を表示するために今すぐブラウザをリロードし
ますか？
```

---

Done.

---

thank you!